Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -31,7 +31,7 @@ public StringAssertions(string value) : base(value)
         /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
         /// </param>
         /// <param name="becauseArgs">
-        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// Zero or more objects to format using the placeholders in <paramref name="because" />.
```

Comment:
Nice finding of this wrong use of `see`.
I would prefer I we changed all 488 wrong uses in a single go, instead of spreading the across PRs.

Reply:
Shall I make a separate issue of it?

Reply:
You can skip creating and issue and go straight to PR.